### PR TITLE
[pwa] Create page of pwa-builder links to demos and components, pwa-l…

### PR DESCRIPTION
**Pull request by vchapel**
…inks

New file per  work item 29745786

### CODE REVIEW

File: microsoft-edge/progressive-web-apps-chromium/pwa-links.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+[Web Share API][pwaWebShareAPI] - Allows you to share URLs and files using the native share UI.
+
+[Web Shortcuts API][pwaWebShortcutsAPI] - Allows you to create a list of shortcuts, or *jumplists*, for your PWA.
+
+## Components
+
+[2D inking on the web][pwa2DInking] - In 2 lines of code add fast, low latency inking canvas with an optional toolbar.
+
+[Agenda Graph Component][pwaAgendaGraphComponent] - Returns calendar events for the signed in user from the Microsoft Graph API.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/pwa-links.md:43]:
```suggestion
[Agenda Graph Component][pwaAgendaGraphComponent] - Returns calendar events for the signed-in user from the Microsoft Graph API.
```

File: microsoft-edge/progressive-web-apps-chromium/pwa-links.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+[Graph Login Component][pwaGraphLoginComponent] - A button and flyout control to facilitate Microsoft identity platform authentication.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/pwa-links.md:45]:
```suggestion
[Graph Login Component][pwaGraphLoginComponent] - A button and flyout control to support Microsoft identity platform authentication.
```
Acrolinx would probably flag "facilitate".

File: microsoft-edge/progressive-web-apps-chromium/pwa-links.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/pwa-links.md:46]: Run Acrolinx on this .md file.

File: microsoft-edge/progressive-web-apps-chromium/pwa-links.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+[Install your PWA][pwaInstallYourPWA] - Gives users a button to install your PWA directly from the browser.
+
+[People Graph Component][pwaPeopleGraphComponent] - Returns the Contacts for the signed in user from the Microsoft Graph API.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/pwa-links.md:49]:
```suggestion
[People Graph Component][pwaPeopleGraphComponent] - Returns the Contacts for the signed-in user from the Microsoft Graph API.
```

File: microsoft-edge/progressive-web-apps-chromium/pwa-links.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+[People Picker Graph Component][pwaPeoplePickerGraphComponent] - Enables the user to view, add, or remove people objects from a dynamic drop-down list as they type.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/pwa-links.md:51]:
```suggestion
[People Picker Graph Component][pwaPeoplePickerGraphComponent] - Enables the user to view, add, or remove **People** objects from a dynamic drop-down list as they type.
```
not sure, needs attention

File: microsoft-edge/progressive-web-apps-chromium/pwa-links.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+[Sign in with Microsoft, Google, or Facebook][pwaSignIn] - Lets your users sign-in using their Microsoft, Google, or Facebook account using `pwa-auth`. Your app receives their email address, name, and profile picture.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/pwa-links.md:53]:
```suggestion
[Sign in with Microsoft, Google, or Facebook][pwaSignIn] - Lets your users sign-in through their Microsoft, Google, or Facebook account by using `pwa-auth`. Your app receives the user's email address, name, and profile picture.
```

File: microsoft-edge/progressive-web-apps-chromium/pwa-links.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+[Tasks Graph Component][pwaTasksGraphComponent] - Returns the tasks for the signed in user from the Microsoft Graph API.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/pwa-links.md:55]:
```suggestion
[Tasks Graph Component][pwaTasksGraphComponent] - Returns the tasks for the signed-in user from the Microsoft Graph API.
```

File: microsoft-edge/progressive-web-apps-chromium/pwa-links.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+ms.prod: microsoft-edge
+keywords: progressive web apps, PWA, Microsoft Edge, PWABuilder, PWA Builder, PWA Builder demos, PWA Builder components
+---
+# Progressive Web App Builder demos and components
+
+This page provides links to Progressive Web App \(PWA\) Builder demos and components.
+
+## Demos
+
+[Background Fetch API][pwaBackgroundFetchAPI] - Enables your web app to fetch large content videos, podcasts, and so on, in the background. Fetch works even when your PWA isn't open.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/pwa-links.md:17]:
```suggestion
[Background Fetch API][pwaBackgroundFetchAPI] - Enables your web app to fetch large content such as videos and podcasts in the background. The `Fetch` API works even when your PWA isn't open.
```

File: microsoft-edge/progressive-web-apps-chromium/pwa-links.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+[Low-latency canvas API][pwaLowLatencyCanvasAPI] - Tells the canvas to skip most of the rendering pipeline in the browser to get pixels on screen with minimal latency.
+
+[Native File System API][pwaNativeFileSystemAPI] - Allows you to read and write files just like a native app.
+
+[Periodic Background Sync API][pwaPeriodicBackgroundSyncAPI] - Allows your app to sync with your server in the background, ensuring your data is always fresh.
+
+[Picture-in-Picture API][pwaPictureInPictureAPI] - Creates a floating always-on-top element to display video sources.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/pwa-links.md:25]:
```suggestion
[Picture-in-Picture API][pwaPictureInPictureAPI] - Creates a floating, always-on-top element to display video sources.
```

File: microsoft-edge/progressive-web-apps-chromium/pwa-links.md (hunk 3 above)
Comment (vchapel) [reply microsoft-edge/progressive-web-apps-chromium/pwa-links.md:46]: Wonder why I didn't? Fixing

**Review by mikehoffms — APPROVED** [review]

**Comment by vchapel** [comment]
Waiting for content update

**Pull request closed without merging**